docusaurus/video/docusaurus/docs/api/gdpr/calls.mdx

```diff
@@ -0,0 +1,85 @@
+---
+id: calls
+sidebar_position: 1
+slug: /gdpr/calls
+title: Calls
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+## Deleting a call
+
+You can either soft-delete or hard-delete a call and all its related data (members, sessions, recordings, transcriptions).
+
+### Soft delete
+
+Soft-delete a call means that the call and all its related data will not be completely removed from our system but will no longer be accessible through the API.  
+The data can still be recovered.
+
+
+<Tabs groupId="examples">
+<TabItem value="js" label="JavaScript">
+
+```js
+// soft-delete a call
+const resp = await call.delete({
+  hard: false,
+})
+// resp.call contains call information
+```
+
+</TabItem>
+<TabItem value="curl" label="cURL">
+
+```bash
+# Soft-delete a call
+curl -X POST "https://video.stream-io-api.com/api/v2/video/call/${CALL_TYPE}/${CALL_ID}/delete?api_key=${API_KEY}" \
+  -H "Authorization: ${TOKEN}" \
+  -H "stream-auth-type: jwt" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "hard": false
+  }'
+```
+</TabItem>
+</Tabs>
+
+
+### Hard delete
+
+Hard-delete a call means that the call and all its related data will be completely wiped out from our system.
+This action is irrevocable, and the data cannot be recovered.
+
+This operation is done asynchronously and you can use the returned `task_id` to monitor its progress.  
+See [how to monitor an async task](/api/misc/async).
```

Comment:
this abs path link is probably not going to work in production (docs are not at the root)